src/aspire/operators/filters.py

```diff
@@ -33,6 +33,30 @@ def wavelength_to_voltage(wavelength):
     ) / (2 * 0.978466)
 
 
+def evaluate_grid_src(src, L, power=1):
```

Comment:
Maybe `evaluate_src_filters_on_grid`? Also, does `L` need to be a separate argument here, can it not be taken from `src`?

Reply:
Technically in the places it's used, the method uses the `L` from the `Estimator`, so I didn't replace it with `src.L`

Reply:
But that `L` itself comes from `src.L` (and should therefore be removed, most likely): https://github.com/ComputationalCryoEM/ASPIRE-Python/blob/3223d6c2c308af66f321ca2b0ddf3933045f904b/src/aspire/reconstruction/estimator.py#L24

Reply:
Oh good point, yes will change.

Reply:
Changed. I think it would be out of scope of this PR to remove the `Estimator.L` overall but I can break it out into an issue?

Reply:
Yes please do.